.prettierignore

```diff
@@ -1,2 +1,7 @@
 # assets contains pre-generated files like iconmoon, fonts, svgs.
 packages/design/src/assets/*
+
+# Ignore file types:
+packages/**/*.css
+packages/**/*.json
+packages/**/*.yaml
```

Comment:
Why do we want to ignore these?

Reply:
we have huge json files used for fixtures that doesn't benefit from formatting, and it also slowed down vscode when viewing the file formatted vs when it wasn't

we also have like yaml template files (like auth connectors, trusted cluster etc), it was adding extra spaces so figured we leave those alone

we probably don't need css here since i ignored the entire directory that contained the pre generated css files from like icomoon, but we can remove it later if we want